src/components/events/createEvent/createEvent.tsx

```diff
@@ -35,7 +35,7 @@ import BasicDetails from './basicDetails';
 
 const CreateEvent = () => {
   const [newEvent, setNewEvent] = useState<IEvent>(defaultRAAMEvent);
-  const [currDates, setCurrDates] = useState<Date[]>([]);
+  const [currDates, setCurrDates] = useState<string[]>([]);
   const [stations, setStations] = useState<RAAM_STATIONS[] | BAZ_STATIONS[]>(
```

Comment:
why the STATIONS aren't just arrays?

Reply:
They are arrays, but of a specific enum.
What's the problem?

Reply:
i dont get why you have array of enums? and not the array of all the stations

if you need an interface you can make typescript interface from array 
```
const RAAM_STATIONS= <const >[ '8R', '8', '8L', 'RCFT '5', 'LCFT '2R', '2', '2L']
 type Raam_stations= typeof RAAM_STATIONS[number]
```

later in `versionConfig` you can use `  sta: RAAM_STATIONS | BAZ_STATIONS;`
and just write '8' in the sta of the interface 
also here you dont need to always use OBJ.values(enum) ,  just use the array

Reply:
Fixed for both stations and computers